generate.sh

```diff
@@ -6,10 +6,12 @@ set -e
 EXAMPLES_GENERATED_DIR=examples/generated
 
 echo "Ensuring we have NPM packages installed..."
-npm install
+# npm install
+yarn
```

Comment:
Why the change from npm to yarn? 

I have a strong (mostly personal) for this project to remain using npm.

Reply:
As mentioned earlier, rules_nodejs recommends it.